speech/snippets/context_classes.go

```diff
@@ -0,0 +1,72 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package snippets contains speech examples.
+package snippets
+
+import (
+	speech "cloud.google.com/go/speech/apiv1"
+	speechpb "google.golang.org/genproto/googleapis/cloud/speech/v1"
+
+	"context"
+	"fmt"
+	"io"
+	"io/ioutil"
+)
+
+// [START speech_context_classes]
+
+// contextClasses provides "hints" to the speech recognizer
+// to favour specific classes of words in the results
+func contextClasses(w io.Writer, gcsURI string) error {
```

Comment:
context is a bit overloaded in go land.  Maybe rename this (and update references) to be clearer that you're talking about speech adaptation or a speech context?

Reply:
Speech unfortunately using both words to describe the same thing: https://cloud.google.com/speech-to-text/docs/speech-adaptation and https://cloud.google.com/speech-to-text/docs/reference/rpc/google.cloud.speech.v1#speechcontext

And I believe they're repurposing the docs to use "Context" throughout, so IMO we should call it contexts throughout. 

I don't know anything about Go though so maybe we need to address this

Reply:
happy to do anything. deferring decision to @czahedi

Reply:
We should keep this region tag as-is because all the docs will be changing to "context".